config/components/webhook/mutating-patch.yaml

```diff
@@ -0,0 +1,26 @@
+- op: replace
```

Comment:
does this override the the configuration for the jobset webhooks? the one named "mutate-jobset-x-k8s-io-v1alpha2-jobset"?

Reply:
Nope, just the mutating webhook configuration at index 1 in the mutating-webhook-configuration `webhooks` array, which is the pod mutating webhook: https://github.com/kubernetes-sigs/jobset/blob/dbabc65129107fd830e7f6b367e9223c87677d7e/config/components/webhook/manifests.yaml#L27-L44

Same goes for the validating-webhook-configuration patch.